website/docs/r/nsxt_app_port_profile.html.markdown

```diff
@@ -69,10 +72,14 @@ The following arguments are supported:
 
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful
   when connected as sysadmin working across different organisations.
-* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level.
+* `vdc` - (Deprecated; Optional) The name of VDC to use, optional if defined at provider level.
+  Deprecated and replaced by `context_id`
+* `context_id` - (Optional) ID of NSX-T Manager, VDC or VDC Group. Replaces deprecated fields `vdc`
```

Comment:
I would like to read more about the term "context" in this context :) Maybe you have some link from documentation?

The reason: the fact that we are using `context_id` here instead of `owner_id` is a bit unexpected for the fresh eyes and I would like to understand justification better.

Reply:
Thanks!

Navigating from the above link to here made it even clearer:
https://developer.vmware.com/apis/vmware-cloud-director/latest/data-structures/NetworkProviderContext/

Also, seeing that in the type it is called `ContextEntityId`:
https://github.com/vmware/go-vcloud-director/blob/main/types/v56/nsxt_types.go#L356

Reply:
So, after more thought, let's keep the `context_id`, because:
- It's called context in API
- Compared to `owner_id` it accepts not only VDC and VCD Group, but also NSX-T Manager - so this field's functionality is a bit different
- UI doesn't operate with neither "context" nor "owner" notion for this resource

<img width="910" alt="image (1)" src="https://user-images.githubusercontent.com/11330689/161269087-30abe057-ee30-4d5b-aa41-6b49f8395063.png">